### ISSUE TITLE
Creating invoice configuration. Configuration should have the numbering with a reset capability. Further it should allow the user to activate or not. Default state, it is not activated.

### ISSUE THREAD

**Issue by bluefin18** [NewEconomyMovement/NEMAndroidApp#12]

**Comment by fyahfox** [comment]
Related to https://github.com/NewEconomyMovement/NEMAndroidApp/issues/17

**Comment by fyahfox** [comment]
I agree that a reset capability would be nice. If that will be implemented we also need an offset which can be defined in the configuration. For example: Start with invoice number 1000 (not 1).

**Comment by EvertonMelo** [comment]
@fyahfox 

What is the status of this, do you need help?

**Comment by EvertonMelo** [comment]
@fyahfox 

What is the status of this, do you need help?

**Comment by fyahfox** [comment]
Sorry, I don't know since I'm not developing any code.

**Comment by EvertonMelo** [comment]
@fyahfox 

What do you think about separating projects?

https://github.com/NemProject/NEMAndroidApp/projects
